gulpfile.js

```diff
@@ -11,9 +11,6 @@ gulp.task('minify-icons', function() {
     .pipe(svgmin({
       plugins: [
         {
-          removeDimensions: true
```

Comment:
By removing this, do we still need to set the `background-size` on a per-icon basis?